etcdserver/server.go

```diff
@@ -857,6 +856,31 @@ func (s *EtcdServer) RemoveMember(ctx context.Context, id uint64) error {
 	return s.configure(ctx, cc)
 }
 
+func (s *EtcdServer) mayRemoveMember(id uint64) error {
+	if !s.Cfg.StrictReconfigCheck {
+		return nil
+	}
+
+	if !s.cluster.IsReadyToRemoveMember(id) {
+		plog.Warningf("not enough started members, rejecting remove member %v", id)
```

Comment:
`%s", types.ID(id))`, so that the log output format can be consistent with other hex ids?